misc-utils/waitpid.1.adoc

```diff
@@ -0,0 +1,47 @@
+//po4a: entry man manual
+= waitpid(1)
+:doctype: manpage
+:man manual: User Commands
+:man source: util-linux {release-version}
+:page-layout: base
+:command: waitpid
+
+== NAME
+
+waitpid - utility to wait for arbitrary processes
+
+== SYNOPSIS
+
+*waitpid* [-v] pid...
```

Comment:
How about changing this to
```
*waitpid* [options] pid...
```
, then...

Reply:
I think for other commands non-functional options like `--help` and `--version` are also not listed in SYNOPSIS